lists/cl/cl-mp.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": {
+    "en": "Public Market Suppliers and Buyer Units, Chile",
+    "local": "Proveedor y Unidad Comprador Mercado Público, Chile"
+  },
+  "url": "https://www.mercadopublico.cl/Home",
+  "description": {
+    "en": "Mercado Público is an electronic platform managed by Chilecompra, part of the Ministry of Finance, which public organizations use to make purchases.\n\nOrganizations registered as buyers or suppliers on the Mercado Público platform are assigned an identifier in the system. This identifier is used in the data from the Mercado Público OCDS API.\n\nThe use of Mercado Público identifiers is considered a temporary measure. They should only be used when no other stable identifier for a government agency is available"
+  },
+  "coverage": [
+    "CL"
+  ],
+  "subnationalCoverage": [],
+  "structure": [
+    "company",
+    "government_agency"
+  ],
+  "sector": [],
+  "code": "CL-MP",
+  "confirmed": false,
+  "deprecated": false,
+  "listType": "local",
+  "access": {
+    "availableOnline": false,
+    "onlineAccessDetails": null,
+    "publicDatabase": "https://apis.mercadopublico.cl/OCDS/data/comprador/unidad/{CodigoOrganismo}",
+    "guidanceOnLocatingIds": "The API accepts an identifier as a parameter and returns the name and taxpayer number ([RUT](http://org-id.guide/list/CL-RUT)) for the organization.\n\nThe `/comprador/` endpoint covers both buyers and suppliers, whilst the [`/proveedor/` endpoint](https://apis.mercadopublico.cl/OCDS/data/proveedor/unidad/{CodigoOrganismo}) covers only suppliers.\n\nThere is no public list of identifiers, however the identifiers are used in Chile's [OCDS API](https://desarrolladores.mercadopublico.cl/docs/services/5a9ed02f718ed712f4403e75/operations/5c07f9b0bffba6b1c82076e8?) which includes links to the appropriate endpoint for each identifier.",
+    "exampleIdentifiers": "2544,2391,26797",
+    "languages": ["es"]
+  },
+  "data": {
+    "availability": [
+      "data_not_available"
+    ],
+    "dataAccessDetails": "",
+    "features": [],
+    "licenseStatus": "open_license",
+    "licenseDetails": ""
+  },
+  "links": {
+    "opencorporates": "",
+    "wikipedia": "https://es.wikipedia.org/wiki/ChileCompra"
```

Comment:
maybe this one too? https://www.mercadopublico.cl/Home/Contenidos/QueEsMercadoPublico?esNuevaHome=true

Reply:
I'm not sure the schema supports links other than to opencorporates and wikipedia, so I've added this link to the list description in https://github.com/org-id/register/pull/424/commits/ca9c0f0eec1e827ccb53925860ca46ecf386ac58